backend/src/docker_tests/dependencies.py

```diff
@@ -9,10 +9,7 @@
 
 def get_docker_client() -> Generator[DockerClient, Any, None]:
     """Creates docker client, which is closed afterwards"""
-    try:
-        client = docker.from_env()
-    except DockerException:
-        raise DockerDeamonNotFound()
+    client = docker.DockerClient(base_url='unix://var/run/docker.sock')
```

Comment:
dit is niet echt ideaal, is het met je usergroups aan te passen gisteren niet opgelost zodat dit niet meer hoeft?

Reply:
nee idd vergeten terugzetten
zou nrml moeten werken